frontend/src/lib/components/ActivationSidebar/activationLogic.ts

```diff
@@ -89,6 +95,34 @@ export const activationLogic = kea<activationLogicType>([
                 setShowSessionRecordingConfig: (_, { value }) => value,
             },
         ],
+        areMembersLoaded: [
+            false,
+            {
+                loadMembersSuccess: () => true,
+                loadMembersFailure: () => true,
```

Comment:
For all of these, if there is a failure do we actually want to say that it's ready? Seems like we'd be better off only showing the thing if things actually load properly.

Reply:
I was thinking we could still show it anyways, even if we fail the calls, as this component informs you about a bunch of features such as session recordings, and it could be informative even if the counts are not correct. But happy to switch it to false, there's not a big difference I think.